src/test/ratings.spec.js

```diff
@@ -0,0 +1,76 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import faker from 'faker';
+import app from '../index';
+
+chai.use(chaiHttp);
+chai.should();
+
+const ratings = {
+  ratings: faker.random.number({
+    min: 1,
+    max: 5
+  })
+};
+describe('PATCH, /api/v1/articles/:articleId/:ratings', () => {
```

Comment:
Good work so far @codexempire but I do not think you need to write a test to validate the functionality of the token as it has already been taken care of previously